src/new-block/blocks/column/edit.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import type { BlockEditProps, TemplateArray } from '@wordpress/blocks';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+import clsx from 'clsx';
+
```

Comment:
We're trying to use the new import style with no internal/external comments

Reply:
✅